.github/workflows/release-prod.yaml

```diff
@@ -275,13 +275,19 @@ jobs:
           APP_CHANNEL: CI
           USES_DEV_BUCKET: "0"
         run: |
+          # re-promote a release containing an old version of embedded-cluster to test upgrades
+          export APP_VERSION="appver-${{ needs.get-tag.outputs.tag-name }}-1.8.0-k8s-1.28"
+          replicated release promote 11615 2cHXb1RCttzpR0xvnNWyaZCgDBP --version "${APP_VERSION}"
+          replicated release promote 11615 2eAqMYG1IEtX8cwpaO1kgNV6EB3 --version "${APP_VERSION}"
```

Comment:
Can we make these some constants so we can easily reason about them ? I mean `11615` and `2eAqMYG1IEtX8cwpaO1kgNV6EB3`.